setup.py

```diff
@@ -30,19 +30,19 @@
 REQUIRED = [
     # simulation
     "attrs",
-    "pyyaml",
-    "numexpr",
-    "numpy>=1.20",
-    "scipy>=1.1",
+    "pyyaml~=6.0",
+    "numexpr~=2.0",
+    "numpy~=1.20",
```

Comment:
This line pins the Numpy version to 1.20, so I think the Numpy version should stay as `>=1.20`. FLORIS requires this minimum version, but it supports greater than this:
```
(floris) >>$ pip freeze | grep numpy
numpy==1.24.2
```

To keep the bounds, I think this will work: `numpy>=1.20,<2`. Since there is no Numpy v2, I wonder if this is valid, but worth trying.

Reply:
Ah shoot I keep misreading the `~=` - disregard this comment